client/src/components/WorkflowInvocationState/WorkflowInvocationState.vue

```diff
@@ -125,7 +125,7 @@ export default {
             return this.invocationSchedulingTerminal && this.jobStatesTerminal;
         },
         invocationLink: function () {
-            return getUrl(`workflows/invocations/report?id=${this.invocationId}`);
+            return getUrl(`page/edit_report?id=${this.invocationId}`);
```

Comment:
Is that correct ?

Reply:
I also don't seem to be able to edit or create a page from an invocation, that would be pretty cool.